change/@ni-nimble-components-dc071140-b97c-473f-b929-1268a3aa3a40.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
What makes this "minor" instead of "patch"?

Reply:
I wasn't sure, but I thought since we were technically exporting a new type (`DialogBase`) it should be "minor". But it's intended as an internal type, so I'm not sure.